templates/pkg/resource/sdk.go.tpl

```diff
@@ -40,52 +43,57 @@ var (
 {{- end }}
 
 // sdkCreate creates the supplied resource in the backend AWS service API and
-// returns a new resource with any fields in the Status field filled in
+// returns a copy of the resource with resource fields (in both Spec and
```

Comment:
We discussed this earlier that currently the spec fields are not updated for sdkCreate. We plan to allow it but currently this godoc is inconsistent with behavior.

"{{ GoCodeSetCreateOutput .CRD "resp" "ko" 1 false }}" -> the false parameter here does not allow spec update.

Reply:
As part of conversation on meeting, this godoc can be updated later.